### PR TITLE
Moving config entries/cleanup

### PR DESCRIPTION
**Pull request by Combie81**
Moved some configutation options to the config files

Minor text changes

Converted a lot of spaces to tabs

### CODE REVIEW

File: firmware_mod/config/telegram.conf.dist [hunk 1]
```diff
@@ -3,5 +3,5 @@
 ############################################################
 
 # Add your Telegram credentials here (these are just examples)
-apiToken="23132160:AAFrSEwzJ-X-64tLnD8JJ$JoAxWIMg$$w"
-userChatId="30985132"
+apiToken="yuorapitoken"
+userChatId="yourchatid"
```
Comment (jmtatsch) [inline firmware_mod/config/telegram.conf.dist:7]: I think those fantasy credentials convey more information about what is expected to be put there. Please leave as it was.

File: firmware_mod/run.sh [hunk 2]
```diff
@@ -268,9 +268,12 @@ done
 
 ## Autostart startup userscripts
 for i in /system/sdcard/config/userscripts/startup/*; do
-  if [ $i != *".dist" ]; then
+  if [[ ${i: -3} == ".sh" ]]; then
```
Comment (jmtatsch) [inline firmware_mod/run.sh:271]: 👍🏻

File: firmware_mod/www/css/camera.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            $('#save_result').html(result);
+        else
+            $('#save_result').html("Nothing to update");
+    });
+    
+}
+
+//Function get config
+function getConfig() {
+    // get config and put to hmtl elements
```
Comment (jmtatsch) [inline firmware_mod/www/css/camera.js:22]: Typo

**Comment by saipsa** [comment]
> You must hate spaces wink

Someone else may do a PR later, converting tabs to spaces. I personally use tabs. 

But tabs to spaces or vice versa PRs should not happen.

**Comment by jmtatsch** [comment]
I personally prefer tabs but don’t have a strong opinion about it. It was a shitload of work to review because so many meaningless changes. Next time better make separate pull requests for minor fixes and reformatting.

**Comment by Combie81** [comment]
Example tokens reverted
Conflicts resolved
Typo's fixed

I moved the config entries as I thought that would allow easier customisation should people prefer different formats/locations and this should make it easier to add the entries to the web gui in the future.

It's not that I hate spaces - I just remembered being taught to use tabs instead of spaces.  It does also bring the file sizes down a few bytes each :) and those space savings add up :p

Apologies for all the review work - I didn't think of that!

**Comment by jmtatsch** [comment]
No worries about reviewing. Thank you for producing something to review :)

**Pull request merged**